src/main/java/com/manardenza/controller/ReservationController.java

```diff
@@ -3,28 +3,50 @@
 import com.manardenza.entity.Hotel;
 import com.manardenza.entity.Reservation;
 import com.manardenza.entity.Room;
+import com.manardenza.login.CurrentUser;
 import com.manardenza.service.ReservationService;
+import org.slf4j.LoggerFactory;
 
 import java.util.Date;
 import java.util.List;
 import java.util.Map;
 
+
 public final class ReservationController {
 
     private ReservationService reservationService;
+    private CurrentUser currentUser;
+    private org.slf4j.Logger log = LoggerFactory.getLogger(HotelController.class);
 
-    public ReservationController(ReservationService reservationService) {
+    public ReservationController(ReservationService reservationService, CurrentUser currentUser) {
         this.reservationService = reservationService;
+        this.currentUser = currentUser;
     }
 
-    public Long bookRoom(Date reservedFrom, Date reservedTo, Room room, Hotel hotel) {
-        return reservationService.bookRoom(reservedFrom, reservedTo, room, hotel);
+    public long bookRoom(Date reservedFrom, Date reservedTo, Room room, Hotel hotel) {
+        log.info(String.format("%s has requested reservation of room %s in hotel %s from %d till %d", currentUser
+            .getUser().getFullName(), room.getId(), reservedFrom, reservedTo));
+        long reservationId = reservationService.bookRoom(reservedFrom, reservedTo, room, hotel);
+        log.info(String.format("%s has successfully reserved room %s in hotel %s from %tD till %tD", currentUser
+            .getUser().getFullName(), room.getId(), reservedFrom, reservedTo));
+        return reservationId;
     }
 
     public boolean cancelReservation(long idReservation) {
-        return reservationService.cancelReservation(idReservation);
+        log.info(String.format("%s has requested cancellation of reservation %d", currentUser
+            .getUser().getFullName(), idReservation));
+        boolean wasCancelled = reservationService.cancelReservation(idReservation);
+        if (wasCancelled) {
+            log.info(String.format("%s has successfully cancelled reservation %d", currentUser
+                .getUser().getFullName(), idReservation));
+        } else {
+            log.info(String.format("Reservation %d cancellation failure", currentUser
+                .getUser().getFullName(), idReservation));
+        }
+        return wasCancelled;
     }
 
+    //TODO: consider deleting this method if it won't be used in console
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)